bitnami/kube-prometheus/templates/exporters/kube-controller-manager/service.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.kubeControllerManager.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager
+  namespace: kube-system
```

Comment:
it should be `{{ .Values.kubeControllerManager.namespace }}`

Reply:
corrected